src/index.ts

```diff
@@ -16,7 +16,7 @@ export * from './common/const';
 export * from './types';
 
 // we have to load these with require so that in the browser side module nothing fails when they aren't included in the webpack bundle
-const client = require('./node/client').DevWorkspaceClient;
+const client = require('./node/client-factory').DevWorkspaceClientFactory;
```

Comment:
Can you try to get rid of `require`s as the client doesn't have a browser-side module?

Reply:
I understand very little why it's here. Contribution is welcome